### PR TITLE
Epmlstrtai 746

### PR DESCRIPTION
**Pull request by stelmakhmaksim**
Added displayedStatus field for correct displaying statuses.

### CODE REVIEW

File: src/main/java/com/epam/test_generator/controllers/caze/response/CaseDTO.java [hunk 1]
```diff
@@ -136,6 +136,10 @@ public void setStatus(Status status) {
         this.status = status;
     }
 
+    public String getDisplayedStatus() {
+        return status.getStatusName();
```
Comment (pomkine) [inline src/main/java/com/epam/test_generator/controllers/caze/response/CaseDTO.java]: Can we avoid using `Status` in this dto and just set `statusName` and `statusDisplayName` values here?

File: src/main/java/com/epam/test_generator/entities/Status.java [hunk 2]
```diff
@@ -9,11 +9,11 @@
  */
 public enum Status {
 
-    NOT_DONE("Not done"),
-    NOT_RUN("Not run"),
-    PASSED("Passed"),
-    FAILED("Failed"),
-    SKIPPED("Skipped");
+    NOT_DONE("NOT DONE"),
```
Comment (hdghg) [inline src/main/java/com/epam/test_generator/entities/Status.java]: I doubt if we need separate getters *getDisplayedName* when *getStatus* returns exact same result.

Comment (stelmakhmaksim) [reply src/main/java/com/epam/test_generator/entities/Status.java]: Not quite, getStatus method returns enum as string (left value) and getDisplayedName returns the string that belongs to enum (right value).

File: src/main/java/com/epam/test_generator/controllers/caze/response/CaseDTO.java (hunk 1 above)
Comment (stelmakhmaksim) [reply src/main/java/com/epam/test_generator/controllers/caze/response/CaseDTO.java]: We can do it. Do you suggest in all response DTO use String instead Status and delete additional statusDisplayName method?

**Pull request merged**